config/routes.rb

```diff
@@ -4,7 +4,7 @@
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :instruments, only: %i[index new create show] do
-    resources :bookings, only: %i[create index]
```

Comment:
You will also need to make changes to the bookings controller, I have sent a modified version, perhaps you can pull down this branch